docs/apis/data_en.md

```diff
@@ -0,0 +1,205 @@
+|-------|----|--------|-----|
+|`data_dir`|`str`|Directory that stores the dataset.||
+|`file_list`|`str`|File list path. file list is a text file, in which each line contains the path infomation of one sample.The specific requirements of `SegDataset` on the file list are listed below.||
+|`transforms`|`paddlers.transforms.Compose`|Data transformation operators applied to input data.||
+|`label_list`|`str` \| `None`|Label list file. label list is a text file, in which each line contains the name of class.|`None`|
+|`num_workers`|`int` \| `str`|The number of auxiliary processes used when loading data. If it is set to `'auto'`, use the following rules to determine the number of processes to use: when the number of CPU cores is greater than 16, 8 data read auxiliary processes are used; otherwise, the number of auxiliary processes is set to half the counts of CPU cores.|`'auto'`|
+|`shuffle`|`bool`|Whether to randomly shuffle the samples in the dataset.|`False`|
+
+The requirements of `SegDataset` for the file list are as follows:
+
+- Each line in the file list should contain two space-separated items representing, in turn, the path of input image relative to `data_dir` and the path of the segmentation label relative to `data_dir`.
+
+## API of Data Reading
+
+Remote sensing images come from various sources and their data formats are very complicated. PaddleRS provides a unified interface for reading remote sensing images of different types and formats. At present, PaddleRS can read common file formats such as .png, .jpg, .bmp and .npy, as well as handle GeoTiff, img and other image formats commonly used in remote sensing.
+
+Depending on the practical demands, the user can choose `paddlers.transforms.decode_image()` or `paddlers.transforms.DecodeImg` to read data. `DecodeImg` is one of [Data transformation operators](#Data transformation operators), can be combined with other operators. `decode_image` is the encapsulation of `DecodeImg` operator, which is convenient use in the way of function calls.
+
+The argument list of `decode_image()` function is as follows:
+
+|Parameter Name|Type|Parameter Description|Default Value|
+|-------|----|--------|-----|
+|`im_path`|`str`|Path of input image.||
+|`to_rgb`|`bool`|If `True`, the conversion of BGR to RGB format is performed. This parameter is not used and may be removed in the future. Do not use it if possible.|`True`|
+|`to_uint8`|`bool`|If `True`, the image data read is quantized and converted to uint8.|`True`|
+|`decode_bgr`|`bool`|If `True`, automatically parses non-geo format images (such as jpeg images) into BGR format.|`True`|
+|`decode_sar`|`bool`|If `True`, single-channel geo-format images (such as GeoTiff images) are automatically parsed as SAR images.|`True`|
+|`read_geo_info`|`bool`|If `True`, the geographic information is read from the image.|`False`|
+|`use_stretch`|`bool`|Whether to stretch the image brightness by 2% linear. Take effect only if `to_uint8` is `True`.|`False`|
```

Comment:
Whether to apply a linear stretch to image brightness (with 2% max and min values removed).